Dockerfile_ubi

```diff
@@ -0,0 +1,41 @@
+ARG BUILD_TAG
+ARG REPO_URL=sumologic-otel-collector
+ARG BUILD_ARCH
+
+FROM alpine:3.19.1 as certs
+RUN apk --update add ca-certificates
+
+FROM alpine:3.19.1 as directories
+RUN mkdir /etc/otel/
+
+## Build RedHat compliant image
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9
+
+ENV SUMMARY="UBI based sumologic-otel-collector" \
+    DESCRIPTION="Sumo Logic Distribution for OpenTelemetry Collector is a Sumo Logic-supported distribution of the OpenTelemetry Collector. It is a single agent to send logs, metrics and traces to Sumo Logic."
+
+LABEL name="sumologic-otel-collector" \
+      vendor="Sumo Logic" \
+      version="$BUILD_TAG" \
+      release="4" \
+      summary="$SUMMARY" \
+      description="$DESCRIPTION" \
+      io.k8s.description="$DESCRIPTION" \
+      maintainer="collection@sumologic.com"
+
+ADD https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/LICENSE \
+    /licenses/LICENSE
+
+RUN microdnf update && microdnf install systemd
+
+ENV TAG $BUILD_TAG
+ARG USER_UID=10001
+USER ${USER_UID}
+ENV HOME /etc/otel/
+
+COPY otelcol-sumo /
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
+COPY --from=directories --chown=${USER_UID}:${USER_UID} /etc/otel/ /etc/otel/
```

Comment:
Why is this directory copied from Alpine and not just created here?

Reply:
see https://github.com/SumoLogic/sumologic-otel-collector/pull/1440#discussion_r1484357187